__tests__/pet.test.js

```diff
@@ -0,0 +1,205 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+    it('returns an object', () => {
+        expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+    
+    it('sets the name property', () => {
+        const pet = new Pet('Fido');
+
+        expect(pet.name).toEqual('Fido');
+    });
+
+    it('has a  initial age of 0', () => {
```

Comment:
Typo:
```
    it('has an initial age of 0', () => {
```

Also, this is a good example of a test that could be undated to use the constants from the other file. You could change the assertion on line 17 to reference the value, and update the name of the test to something like 'initialises with the correct age value'